packages/plugin/src/Integrations/PaymentGateways/PayPal/Controllers/OrdersController.php

```diff
@@ -0,0 +1,110 @@
+<?php
+
+namespace Solspace\Freeform\Integrations\PaymentGateways\PayPal\Controllers;
+
+use Solspace\Freeform\Integrations\PaymentGateways\PayPal\Services\PayPalOrderService;
+use Solspace\Freeform\Integrations\PaymentGateways\PayPal\Services\PayPalPriceService;
+use Solspace\Freeform\Library\Helpers\IsolatedTwig;
+use yii\web\NotFoundHttpException;
+use yii\web\Response;
+
+class OrdersController extends BasePayPalController
+{
+    public $enableCsrfValidation = false;
+    protected array|bool|int $allowAnonymous = true;
+
+    public function __construct(
+        $id,
+        $module,
+        $config,
+        private IsolatedTwig $isolatedTwig,
+        private PayPalOrderService $orderService,
+        private PayPalPriceService $priceService,
+    ) {
+        parent::__construct($id, $module, $config);
+    }
+
+    public function actionCreate(): Response
+    {
+        try {
+            [$form, $integration, $field, $hash] = $this->getRequestItems(handleFormRequest: false);
```

Comment:
`handleFormRequest` is `false` by default already

```suggestion
            [$form, $integration, $field, $hash] = $this->getRequestItems();
```